registrations/tasks.py

```diff
@@ -392,6 +391,48 @@ def run(self, registration_id, **kwargs):
 validate_subscribe = ValidateSubscribe()
 
 
+@app.task()
+def remove_personally_identifiable_fields(registration_id):
+    """
+    Saves the personally identifiable fields to the identity, and then
+    removes them from the registration object.
+    """
+    registration = Registration.objects.get(id=registration_id)
+
+    fields = set((
+        'id_type', 'mom_dob', 'passport_no', 'passport_origin', 'sa_id_no',
+        'language', 'consent')).intersection(registration.data.keys())
```

Comment:
Possibly, but it seems to be a fairly high risk item to keep, looking at our security research, so I opted to take it out.

We can always get that data back by looking up the identity, unless they've opted out, in which case we don't want to use them for the investigation anyway.